### DISCUSSION TITLE
How to use --file-root correctly ?

### DISCUSSION THREAD

**Discussion by bambozi** [Orange-OpenSource/hurl#2369] [Q&A] [answered]
[Options]
delay: 500
very-verbose:true
**file-root:../test-data/**  
and i get the options is not valid 
^ the option name is not valid. Valid values are aws-sigv4, cacert, cert, compressed, connect-to, delay, insecure, http1.0, http1.1, http2, http3, ipv4, ipv6, key, location, max-redirs, output, path-as-is, proxy, resolve, retry
, retry-interval, skip, unix-socket, variable, verbose, very-verbose

**Comment by fabricereix** [comment]
Hi @bambozi 
One of the main purpose of the `--file-root` option is security. 
It must be changed only at the command-line, not from the Hurl file itself.

**Reply by bambozi** [reply]
Hi @fabricereix , thank you for the info, but how can use it in command-line , 
i try this but it didn't work 
[Options]
delay: 500
very-verbose:true
file, **--file-root:../test-data/** ; file.xml ;

**Comment by fabricereix** [comment, marked as answer]
This is a standard command-line option
https://hurl.dev/docs/manual.html#file-root
```
hurl --file-root ../test-data test.hurl
```

**Reply by bambozi** [reply]
@fabricereix Thanks! it work!
